apps/src/BlockStaticAnalyzer.js

```diff
@@ -0,0 +1,408 @@
+/* jshint
+strict: true,
+curly: true,
+funcscope: true,
+newcap: true,
+nonew: true,
+shadow: false,
+unused: true,
+
+maxlen: 90,
+maxparams: 3,
+maxstatements: 200
+*/
+/* global -Blockly */
```

Comment:
Yes; this line *unsets* Blockly as a global for this file, which we have to do since it's still allowed in the global JSHint settings.